src/board/board.ts

```diff
@@ -1,26 +1,31 @@
 import chalk from "chalk";
 import { renderTile, Tile } from "../tile/tile";
+import { checkIfMoveIsAllowed } from "../move/move";
 
 export type NullableTile = Tile | undefined;
 export type Board = NullableTile[][];
 
 const BLANK_CHAR = chalk.black(" ");
+let isFirstRender = false;
 
-export const renderLine = (lines: Tile[]): string => {
+export const renderLine = (
+  lines: NullableTile[],
+  lineIndex: number
+): string => {
   const line = "";
+  let x = 0;
   return lines.reduce((accumulator, tile) => {
     if (tile == undefined) {
       return accumulator + BLANK_CHAR;
     }
 
+    x++;
     return accumulator + renderTile(tile);
   }, line);
 };
 
 export const renderBoard = (data: Board) => {
-  console.log(chalk.white.bold("-------------"));
-  data.forEach((lines) => {
-    console.log(renderLine(lines));
+  data.forEach((lines, y) => {
```

Comment:
You removed the `console.log(chalk.white.bold("-------------"));` instructions. Is this intentional? :thinking:

Reply:
No, it should be okay now !

Reply:
You kept only one out of the two, is this intentional?

Reply:
Nope, it's back !